types/mem-cache/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for mem-cache 1.0
+// Project: https://github.com/silviom/node-mem-cache
+// Definitions by: Pedro Mutter <https://github.com/MutterPedro>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { EventEmitter } from 'events';
+
+export interface CacheOptions {
+    timeout?: number;
+    doesNotRenewTimeout?: boolean;
+    timeoutDisabled?: boolean;
+}
+
+export default class Cache extends EventEmitter {
```

Comment:
It's not `export default`: https://github.com/silviom/node-mem-cache/blob/master/index.js#L276